typescript/widget-store/README.md

```diff
@@ -1,56 +1,35 @@
-# Fault-Tolerant Checkout
+# Widget Store
 
-In this example, we use DBOS and Fastify to build an online storefront that's resilient to any failure.
+This app uses DBOS to build an online storefront that's resilient to any failure.
+You can interrupt it at any time (we even provide a crash button to facilitate experimentation) and it will recover from exactly where it left off.
 
-### Developing Locally
+## Setup
 
-First, set the `DBOS_DATABASE_URL` environment variable to a connection string to your Postgres database.
-(The default `DBOS_DATABASE_URL` for this demo is `postgresql://postgres:dbos@localhost:5432/widget_store_node`.)
-
-Then, install dependencies, build your app, and set up its database tables:
+1. Install dependencies and build the application.
 
 ```shell
 npm install
 npm run build
```

Comment:
should use prelaunch/launch to simplify first run experience while still being compatible w/ DBOS CLoud